dist/index.js

```diff
@@ -55,6 +56,10 @@ function terraform(params) {
     core.setFailed(`Sanity checks failed. Unknown value for 'terraform_lock': ${terraformLock}`);
     process.exit(1);
   }
+  if (/^\d+$/.test(terraformParellelism)) {
+    core.setFailed(`Sanity checks failed. Non-integer value for 'terraform_parallelism': ${terraformParallelism}`);
+    process.exit(1);
+  }
```

Comment:
For an upcoming iteration: perform all sanity checks before actually bailing out.